pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -269,6 +272,7 @@ def setup_distributed(self):
         # set the ranks and devices
         self.dist.rank = self.global_rank
         self.dist.device = self.root_device
+        self._rank_0_has_called_call_children_scripts = self.broadcast(self._rank_0_has_called_call_children_scripts)
```

Comment:
I tested the branch and ran into an error here, which reminded me we can't broadcast until the model is wrapped into DDP, so we must make this call after `configure_ddp()` was called. It's a weird PyTorch mechanic I never found out why it requires that